### DISCUSSION TITLE
Theme inheritance cascade ?

### DISCUSSION THREAD

**Discussion by tsaquet** [marp-team/marp#329] [Q&A] [answered]
Hello !

Thank you for your work on Marp, it is a great tool !
I made different theme inherited from the scss base themes, it works like a charm. 

Now I am trying to inherit from one of my themes to change the format (I'd like one A4 printable format), so I tried `@import 'my_theme'` and `@import-theme 'my_theme'` but it does not seem to work, the only property I have are the one for my new themes, the A4 format, everything else is lost. 

Is it something I do wrong ? A limitation ? A bug ? Thank you for your help !
Thomas

**Comment by yhatt** [comment, marked as answer]
Check all of custom themes that you are tried to import are recognizing by the Marp tool you're using.

```css
/* my_theme.css */

/* @theme my_theme */
/* @size 16:9 1280px 720px */

@import 'default'; /* Import built-in theme */
```

```css
/* my_theme_a4.css */

/* @theme my_theme_a4 */
/* @size A4 210mm 297mm */

@import 'my_theme'; /* Import the custom theme */
```

For example, `my_theme_a4` is depending to `my_theme` in this case. You have to teach the location of both of themes to Marp tool.

For Marp for VS Code:

```jsonc
// .vscode/settings.json
{
  "markdown.marp.themes": [
    "my_theme.css",
    "my_theme_a4.css"
  ]
}
```

For Marp CLI:

```console
marp --theme-set my_theme.css my_theme_a4.css -- your-markdown.md
```

**Reply by tsaquet** [reply]
Thank you for your help :) I finally managed to do what I needed thanks to your message (and reading the doc again !)
